### ISSUE TITLE
 Discord did not accept a text message: TypeError: Cannot read property 'send' of undefined

### ISSUE THREAD

**Issue by KittyCatGamersDiscord** [TediCross/TediCross#96]
![image](https://user-images.githubusercontent.com/34783778/50828366-f21f5680-138c-11e9-8812-35715a587d9e.png)

**Comment by Suppen** [comment]
This is almost always due to a wrong Discord channel ID in the settings. Could you check those? Remember the quotes

**Comment by KittyCatGamersDiscord** [comment]
Just checked them, there all correct

**Comment by Suppen** [comment]
The error message appears because the Discord bot cannot find a channel with the given ID.

It could be that the bot is not in the channel, but considering you got the channelId and serverId (presumably), I guess it is? Unless you got them from some other source than `/chatinfo`

It could potentially also be that the bot does not have sufficient rights in the channel, but I believe the error message would be different then

**Comment by asignificantlackof** [comment]
i have the same issue... but some more information to add.

I've checked and saved the correct IDs for my discord server and the channel i want TediCross to post into at least three times. 

I've verified that I've saved the settings.yaml file correctly, and saved a backup for completeness.

when i start TediCross it logs in correctly to both Telegram and Discord, but as soon as a message comes across from Telegram, I get the above error. 

when i cancel out of TediCross, my settings.yaml file is overwritten, at which point the Discord Server and Channel IDs are changed, and saved incorrectly.

I'm running TediCross on linux (mint) for reference.
